move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -94,6 +93,7 @@ void move_group::MoveGroupMoveAction::executeMoveCallback_PlanAndExecute(const m
 
   if (planning_scene::PlanningScene::isEmpty(goal->planning_options.planning_scene_diff))
   {
+    context_->planning_scene_monitor_->syncSceneUpdates();
```

Comment:
I cannot follow you on this. Could you explain in more detail?
When distributing `syncSceneUpdates()` throughout the code, I mainly looked for instances of `LockedPlanningSceneRO`, which indicated existing use of a (hopefully up-to-date) planning scene.